### PR TITLE
Do not enable elemental-system-agent.service

### PR DESCRIPTION
**Pull request by davidcassany**
We should prevent having enabled services within the image unless we are fully confident we want them to be always enabled and running. Having services enabled just for the bootstrap phase is risky IMHO, as this is written within the immutable filesystem, we should only call the service when needed without enabling it.

Signed-off-by: David Cassany <dcassany@suse.com>

### CODE REVIEW

**Comment by davidcassany** [comment]
This is related to rancher/elemental-operator#86

File: framework/files/system/oem/99_elemental_system_agent.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+name: "Elemental system agent bootstarp"
+stages:
+  network.after:
+    # run only if on live cd and there is a config file
```
Comment (Itxaka) [inline framework/files/system/oem/99_elemental_system_agent.yaml:4]: coment is wrong :D

File: framework/files/system/oem/99_elemental_system_agent.yaml [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+    - if: '[ ! -f /run/cos/live_mode ] && [ ! -f /etc/systemd/system/rancher-system-agent.service ]'
```
Comment (Itxaka) [inline framework/files/system/oem/99_elemental_system_agent.yaml:5]: what about recovery? shouldnt we just launch this under active partition?

**Comment by kkaempf** [comment]
This needs to be followed-up in https://build.opensuse.org/package/view_file/isv:Rancher:Elemental/systemd-presets-branding-Elemental/50-default-Elemental.preset?expand=1

File: framework/files/system/oem/99_elemental_system_agent.yaml (hunk 2 above)
Comment (davidcassany) [reply framework/files/system/oem/99_elemental_system_agent.yaml:5]: good point

**Review by Itxaka — APPROVED** [review]
Nice!

**Comment by davidcassany** [comment]
> This needs to be followed-up in https://build.opensuse.org/package/view_file/isv:Rancher:Elemental/systemd-presets-branding-Elemental/50-default-Elemental.preset?expand=1

Here it is https://build.opensuse.org/request/show/992532

**Pull request merged**